hal/inc/wlan_hal.h

```diff
@@ -413,8 +428,9 @@ int wlan_restart(void* reserved);
 int wlan_set_hostname(const char* hostname, void* reserved);
 int wlan_get_hostname(char* buf, size_t buf_size, void* reserved);
 
-int wlan_set_country_code(const char * country_code);
-int wlan_get_country_code(char * country_code);
+wlan_country_code_t wlan_get_country_code(void* reserved);
```

Comment:
Return `int` here, so that not only `wlan_country_code_t` can be returned, but also `system_error_t`. See above as an example.